interview-questions.md

```diff
@@ -0,0 +1,66 @@
+# ASSESSMENT 2: Interview Practice Questions
+
+Answer the following questions.
+
+First, without external resources. Challenge yourself to answer from memory as if you were in a job interview.
+
+Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
+
+1. What is the difference between a parameter and an argument?
+
+Your answer: The Difference between a parameter and an argument is that, a parameter is a listed name in a function. An argument is  value that passes to the function.
+
+Researched answer:
+Function parameters are the names listed in the function's definition. Function arguments are the real values passed to the function. Parameters are initialized to the values of the arguments supplied.
+
+
+2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
+
+Your answer:  Javascript takes in two  predefined parameters which are considered the arrays, which are great for simplyfining code.
+
+Researched answer: 
+map() creates a new array from calling a function for every array element. map() calls a function once for each element in an array. map() does not execute the function for empty elements. map() does not change the original array.
```

Comment:
This is a great definition of .map, make sure though that you are answering the initial question to the best of you abilities with the researched answer.  In this case, the predefined parameters that map takes is currentValue, index and array with currentValue being required and index and array are optional.